kube-eagle/templates/ingress.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "kube-eagle.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
I think they have migrated from extensions/v1beta1 to networking.k8s.io/v1beta1 now

Reply:
I don't really understand how you want it to be. This PR is for supporting only 1.16 onwards, I will change there too the version to : networking.k8s.io/v1beta1

Reply:
To my knowledge there's no `extensions/v1` api. They are migrating to `networking.k8s.io/v1beta1` in Kubernetes 1.16 (which has been introduced in Kubernetes v1.14). `extensions/v1beta1` will be deprecated with 1.20+, that's why I suggested to leave the apiVersion like it was until 1.20 is actually released.

See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#ingress-v1beta1-networking-k8s-io and https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#ingress-v1beta1-extensions